examples/creating-tables-with-query/creating_tables_with_query.bal

```diff
@@ -14,7 +14,7 @@ public function main() {
     ];
 
     // The query expression starts with `table`.
-    // The key specifier `key(id)` specifies the key sequence of the constructed `table`.
+    // The key specifier `key(id)` specifies the key constraint of the constructed `table`.
```

Comment:
Is this correct? Doesn't constraint apply to table type descriptors while sequence applies to constructed tables?

e.g., https://ballerina.io/spec/lang/master/#table-constructor-expr

@gimantha, thoughts please?